recipes-kernel/linux/linux-fslc/defconfig

```diff
@@ -162,6 +162,7 @@ CONFIG_WL18XX=m
 CONFIG_WLCORE_SDIO=m
 # CONFIG_WILINK_PLATFORM_DATA is not set
 CONFIG_INPUT_EVDEV=y
+CONFIG_INPUT_EVBUG=m
```

Comment:
This is problematic; please drop it or add it to the loading blacklist.

Reply:
Oh, this change was not intended but somehow slipped through.

Thanks for pointing it out, would drop and send a _V2_ now.